client/layout/transient-notices/style.scss

```diff
@@ -3,15 +3,14 @@
 .woocommerce-transient-notices {
 	position: fixed;
 	bottom: $gap-small;
-	left: 0;
+	left: 176px;
```

Comment:
This seems to have affected the OBW as well, is that intentional?

Reply:
And when the menu is collapsed:
![image](https://user-images.githubusercontent.com/224531/84721236-88fd3300-afc3-11ea-982a-79c1bb4605e8.png)
Fixing these would be a little tricky though, they're a little edge-casey, and aren't a big deal so you could probably leave them for a follow-up.

Reply:
I think so, but this was under the Task List section of the issue.

>Snackbar styling and positioning should match Gutenberg.

I think its OK, we can always revert if need be.

Reply:
Cool